src/2_RTB/PianoDiQualifica/content.typ

```diff
@@ -578,15 +637,123 @@ table(
       columns: (80pt,80pt),
       align: (center,center),
       fill:(_,row) =>if row==0 {luma(150)} else if calc.odd(row) { luma(220)} else {white},
-      [*Codice test*],[*Codice caso d'uso*],
+      [*Codice test*],[*Codice Requisito*],
       ..test_di_sistema.map(item => (item.at(0),item.at(3))).flatten().map(item => [#item])
 ),caption: "Tracciamento dei test di sistema.")
 
 
+
+
+#pagebreak()
+= Cruscotto delle metriche
+
+== Qualità di Processo - Fornitura
+
+=== Estimated at Completion
+ 
+#figure(
+  image("./assets/valutazione-metriche/EAC.png", width: 80%),
+  caption: [
+    Valutazione Estimated At Completion.
+  ],
+)
+
+L'EAC rappresenta una revisione del valore stimato per la realizzazione del progetto, ossia il BAC (Budget At Completion) rivisto allo stato corrente del progetto; il fattore che incide maggiormente sull'andamento dell'EAC è il rapporto tra EV (Estimated Value) e AC (Actual Cost), per cui tanto più queste metriche sono vicine l'una all'altra, tanto più l'EAC risulterà vicino al BAC pianificato inizialmente. 
+
+
+Al termine del settimo sprint il team ha rimodulato il BAC progettuale. La modifica non è dovuta tanto dai risultati emersi dalle metriche sottostanti, piuttosto da alcune considerazioni errate in sede di candidatura. 
```

Comment:
```suggestion
Al termine del settimo #glossary[sprint] il team ha rimodulato il BAC progettuale. La modifica non è dovuta tanto dai risultati emersi dalle metriche sottostanti, piuttosto da alcune considerazioni errate in sede di candidatura. 
```

sprint è termine di glossario, quindi va marcato come tale anche in tutte le occorrenze successive.